cypress/helpers/stories.ts

```diff
@@ -0,0 +1,33 @@
+declare global {
+  interface Window {
+    /**
+     * Load a story. This will invoke the storybook router,
+     * unmount a previous story, mount the current story and force a rerender
+     * This should be in a `beforeEach` block to force a fresh new story
+     * @param story Name of the story found in the `.add` function
+     * @param variation Variation of the story found in the `.add` function
+     */
+    setCurrentStory(story: string, variation: string);
+  }
+}
+
+/**
+ * Load a story. This will invoke the storybook router,
+ * unmount a previous story, mount the current story and force a rerender
+ * This should be in a `beforeEach` block to force a fresh new story
+ * @param story Name of the story found in the `.add` function
+ * @param variation Variation of the story found in the `.add` function
```

Comment:
Nit: From the convo yesterday, can we call this a `variant`?

Reply:
Oh, definitely. I'm sorry I missed that

Reply:
I went with `categorization` and `story` with more clear descriptions